lib/plugin-support/shared-worker-loader.js

```diff
@@ -112,6 +121,14 @@ const nextMessageId = () => `${messageIdPrefix}/${++messageCounter}`;
 
 function publishMessage(testWorker, data, replyTo) {
 	const id = nextMessageId();
+	if (testWorker.sharingMap.size) {
```

Comment:
I am assuming `data` is an object and overriding the previously shared buffers in the payload. I am not sure this works when finally serializing `data` with `v8`.